zfmt/ast.go

```diff
@@ -518,6 +518,22 @@ func (c *canon) proc(p ast.Op) {
 			c.space()
 			c.exprs(p.Args)
 		}
+	case *ast.Load:
+		c.next()
+		c.write("load %s", p.Pool)
+		if len(p.Branch) != 0 {
```

Comment:
The conventional way to check for a non-empty string is `s != ""` rather than `len(s) != 0`.
```suggestion
		if len(p.Branch) != 0 {
```